client/src/components/Download.js

```diff
@@ -110,8 +110,9 @@ export const Download = ({ icon, resource }) => {
             <DownloadToken />
           ) : !publicComputedFile ? (
             <DownloadOptions
-              handleSelectFile={handleSelectFile}
+              resource={resource}
               computedFiles={resource.computed_files}
```

Comment:
**Suggestion:** since we're now passing the `resource`, can we remove the line `computedFiles={resource.computed_files}` and destructure the `computed_files` value in `DownloadOptions` instead? 
(Or we'll keep it for the sake of readability?)

Reply:
yeah thats a good call, done!